### PR TITLE
Updated the tox and travis configs -- BOM-1575

### PR DESCRIPTION
**Pull request by mraarif**
Relevant JIRA issue [here](https://openedx.atlassian.net/browse/BOM-1575).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from setuptools import setup
 
 
-version = '1.0.2'
+version = '1.1.0'
```
Comment (awais786) [inline setup.py:8]: Why not 1.0.3 ?

Comment (mraarif) [reply setup.py:8]: dropping older Django versions and adding python 3.8 seemed a significant change.

File: .travis.yml [hunk 2]
```diff
@@ -5,29 +5,26 @@ addons:
...
-  - TOX_ENV=pylint
+- TOX_ENV=django22
+- TOX_ENV=csslint
+- TOX_ENV=eslint
+- TOX_ENV=pycodestyle
+- TOX_ENV=pylint
 before_install:
-  - "pip install -U pip"
-  - export BOTO_CONFIG=/dev/null
+- pip install -U pip
```
Comment (morenol) [inline .travis.yml:17]: maybe we should use the 20.0.2 version here, just like the other PRs

Comment (mraarif) [reply .travis.yml]: done

**Pull request merged**